nucypher/cli/actions/configure.py

```diff
@@ -50,6 +50,23 @@ def forget(emitter: StdoutEmitter, configuration: CharacterConfiguration) -> Non
     emitter.message(SUCCESSFUL_FORGET_NODES, color='red')
 
 
+def get_config_filepaths(config_class: Type[CharacterConfiguration], config_root: str = None) -> List:
+    #
+    # Scrape disk for configuration files
+    #
+    config_root = config_root or DEFAULT_CONFIG_ROOT
+    default_config_file = glob.glob(config_class.default_filepath(config_root=config_root))
```

Comment:
If it's a single file, why use `glob`?

Reply:
code from before that I didn't want to change.